spec/convection/control/stack_spec.rb

```diff
@@ -36,6 +36,28 @@ module Convection::Control
         azs.uniq!
         expect(azs.size).to be(1)
       end
+
+      it 'availability_zones can be ignored' do
+        subject.exclude_availability_zones = %w(eu-central-1a)
+        expect(subject.availability_zones).to contain_exactly('eu-central-1b')
```

Comment:
It may be worthwhile to add a comment here that if Amazon adds an eu-central-1c, this test will break and need to be fixed.  That should save a little time/headache if it does happen.